DEVELOPMENT.md

```diff
@@ -6,6 +6,7 @@ Development is only possible on a (recent) Linux or MacOS machine. Other operati
 
 - Download/clone the repo to your local machine.
 - Set-up the development environment: `scripts/setup.sh`
+- Activate the new Python environment  `source .venv/bin/activate`
```

Comment:
this should be included in the setup script so if that is not the case, we should adjust that

Reply:
Yeah, its already part of the setup script

Reply:
Basically you should ALWAYS run the setup script to prepare your dev environment